<!-- This launch file demonstrates the use of xtion ranger with rtabmap -->
<launch>
	<arg name="map_folder" value="maps" />
	
	<node pkg="ranger_extra" name="mapper" 			type="mapper_app.py" output="screen">
		<param name="path_map"  value="$(find ranger_extra)/$(arg map_folder)"/>
		<param name="path_waypoint" value="$(find ranger_extra)/$(arg map_folder)"/>
		<param name="file_map" value="map"/>
		<param name="file_waypoint" value="waypoints.txt"/>
		<!-- app topic names -->
		<param name="app_topic_frame_add" value="frame_publisher_topic"/>
		
	</node>
</launch>
